aio/aio-proxy/aio_proxy/helpers.py

```diff
@@ -2,13 +2,13 @@
 
 
 def serialize_error_text(text: str) -> str:
-    """Serialize a text string to a JSON formatted string"""
+    """Serialize a text string to a JSON formatted string."""
     message = {"erreur": text}
     return json.dumps(message)
 
 
 def hide_fields(search_result: list) -> list:
```

Comment:
You could put `serialize_error_text`and `hide_fields` in separate files as the first one is related to error management and the second one is related to search